testtools/iothub_test/src/iothub_account.c

```diff
@@ -1137,3 +1149,46 @@ const IOTHUB_MESSAGING_HANDLE IoTHubAccount_GetMessagingHandle(IOTHUB_ACCOUNT_IN
     return result;
 }
 
+IOTHUB_GATEWAY_VERSION IoTHubAccount_GetIoTHubVersion(IOTHUB_ACCOUNT_INFO_HANDLE acctHandle)
```

Comment:
Do we need this check and the caller of it?  What's harm of running same tests against GWv1+v2, other than I guess some extra overhead?

Reply:
Oh - I just saw the 3500 message thing, which is huge and I guess to be avoided.  I'll start another thread on that.